reference/5.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -521,8 +521,13 @@ You can also create ranges in reverse order.
 ```
 
 The start and end values of the range can be any pair of expressions that
-evaluate to an integer. For example, you could use the members of an
-enumeration for your start and end values.
+evaluate to an integer or a character. The endpoints of the range must be
+convertible to signed 32-bit integers (`[int32]`). Larger values cause an
+error. Also, if the range is captured in an array, the count of resulting is
```

Comment:
Thanks for the quick turnaround, but the word "numbers" is presumably missing after "count of resulting", and "the" is missing before "maximum size".

Reply:
Doh!